### ISSUE TITLE
Add rule against calling emergency shuttle wantonly

### ISSUE THREAD

**Issue by ParagonPrime** [yogstation13/yogstation-classic#722]
### Expected Behaviour

Emergency shuttle is a last-ditch safety measure for when shit is really falling apart.
### Actual Behaviour

One person dies/a single redsuit is spotted/bar gets blown up; this is apparently enough of an emergency to stop THE ENTIRE SHIFT OF THIS TOP-OF-THE-LINE NANOTRASEN STATION AND SEND THE WHOLE CREW HOME, WHICH COSTS TIME, MONEY, FUEL, AND LOST WORK EFFICIENCY
### Steps to reproduce the problem
1. Slip Captain on banana peel
2. Shuttle gets called

---

Basically, just add a minor rule saying you should be a little more sensible about when you call the emergency shuttle- NanoTrasen would be rightly pissed if you showed up back at CentCom and your reason for evacuating your workplaces was "well we saw a dude in a Syndicate hardsuit outside the window". Too many easily fixed problems lead to shuttle calls that everyone's too lazy to recall even with extreme public demand.

**Comment by Super3222** [comment]
SS13 is actually regarded as Nanotrasens worst research facility.

**Comment by ToveyAyyLmao** [comment]
I rarely get it seen called due to really dumb reasons. Most times the crew are quite happy to fight whatever is coming. I will admit you get it called for dumb reasons 1:30 minutes into the shift sometimes but thats the traitors faults for making the shift for boring.

**Comment by FluffySurvivor** [comment]
:-1:
We don't need this. Forcing round stalling for the sake of MUH ARGHPEEE ? no.

**Comment by Alek2ander** [comment]
1.4: Do not powergame. As a regular crew member, you're not an experienced soldier. Do not rush into danger without a good reason to do so. Do not collect dangerous items, weapons, rare tools "just in case". Use reasonable self-defense; do not try to kill your attacker in revenge, especially after conflict has already ended. **Do not call the escape shuttle without a proper reason, and do not recall it needlessly.** This rule does not apply to antagonists.

**Comment by ShadowDeath6** [comment]
buttmad_antag.wav

Also a chunk of the station being disintegrated by a bomb is pretty much "shit falling apart"

**Comment by ParagonPrime** [comment]
less buttmad antag, more buttmad... any role that requires time to work on fun stuff, basically.

so scientist, geneticist, virologist, engineer...

anyways it's not round stalling, it's "don't call the fucking shuttle when the crew can handle the fucking situation"

**Comment by Time-Green** [comment]
I would love this.
You have no idea how annoying it is when you almost get fucking x-ray and BoH's and cap calls shuttle because clown slipped him.

**Comment by FluffySurvivor** [comment]
Let's say 50% of pop dies from mass bombing, a few robust engineers manage to fix all the hull breaches and basically everyone has to wait in a stalled as fuck round because "dont u dare call the shuttle" 
BS. 

Recalling stupid shuttle calls as a non command role is not against the rules, so start doing it.

**Comment by DavidEdwards** [comment]
Unnecessary. As Alek points out, we already have a rule for this.
